### ISSUE TITLE
PyTorch 0.4 support

### ISSUE THREAD

**Issue by abhishek0318** [howardyclo/pytorch-seq2seq-example#3]
Thank you for so much for sharing this. Such detailed code on batched sequence to sequence model is not available elsewhere.

With PyTorch 0.4 released, you could move the code to PyTorch 0.4.

**Comment by howardyclo** [comment]
@abhishek0318 Yeah, I faced the same situation as you did, so I wrote this code to help others like you. Very glad to hear that you like it. Since I've been busy for doing another project, maybe I'll find some time to update my codebase. Thanks! :-)
